src/animation/customGraphicTransition.ts

```diff
@@ -221,7 +221,7 @@ export function applyLeaveTransition(
         if (leaveToProps) {
             // TODO TODO use leave after leaveAnimation in series is introduced
             // TODO Data index?
-            const config = getElementAnimationConfig('update', el, elOption, animatableModel, 0);
+            const config = getElementAnimationConfig('leave', el, elOption, animatableModel, 0);
```

Comment:
It's not a typo. Because we haven't introduce `leave` animation config for series yet. We still keep using `update` animation config here for compatitable reason.

Reply:
oh I see, my bad then!